src/edu/wright/cs/raiderplanner/controller/TaskController.java

```diff
@@ -416,7 +416,7 @@ protected boolean computeValue() {
 			this.weighting.setText(Integer.toString(this.task.getWeighting()));
 			this.taskType.getSelectionModel().select(this.task.getType().getName());
 			this.deadline.setValue(this.task.getDeadlineDate().toInstant()
-					.atZone(ZoneId.systemDefault()).toLocalDate());
+						.atZone(ZoneId.systemDefault()).toLocalDate());
```

Comment:
Please revert the change in indentation here. The continuation line should only be indented two tabs beyond the starting line.